delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/postgres/Doobie.scala

```diff
@@ -34,9 +36,41 @@ object Doobie {
     transactors(PostgresContainer.resource(user, pass), user, pass)
 
   trait Fixture { self: NexusSuite =>
-    val doobie: catseffect.IOFixture[Transactors] =
+
+    val doobie: IOFixture[Transactors] =
       ResourceSuiteLocalFixture("doobie", resource(PostgresUser, PostgresPassword))
 
+    /**
+      * Truncate all tables after each test
+      */
+    val doobieTruncateAfterTest: IOFixture[Transactors] = new IOFixture[Transactors]("doobie") {
```

Comment:
shame you have to manually roll this, would be nice to be able to configure the fixture more granularly